utilities/load-test/loadtest/loadtest_test.go

```diff
@@ -413,3 +431,101 @@ func TestAllApplication(t *testing.T) {
 	}
 	t.Log("Prometheus, along with 20 guestbook applications are created, deployed and deleted successfully!")
 }
+
+func TestNResource(t *testing.T) {
+	cleanup := func() {
+		// To delete a application from the given (ex: argocd) namespace and delete the test-namespace created
+		for i := 1; i < 101; i++ {
```

Comment:
New requirement: can you add a constant at the top of the method, like:
```
numberOfAppsToCreate := 100
```
And then use this value in the various for loops? That we can easily control the number of applications to create as part of this test.

Reply:
yes, it will clean the code as well by creating a better understanding thanks! :")